doc/version_history.rst

```diff
@@ -2,6 +2,11 @@
 Version History
 ===============
 
+v0.4.5
```

Comment:
I think these are quite substantial changes for a patch release. Consider making this `v0.5.0`.... Patch releases are mostly for backward compatible bug fixes.

Reply:
Fixed in the commit: [449e880](https://github.com/lsst-ts/ts_m2com/pull/10/commits/449e8806e72c01fd253fbbdfc896198514aad657). Thanks!